src/ALZ/Assets/alz-bicep-config/v0.14.1-pre.config.json

```diff
@@ -23,12 +23,12 @@
             "destination": "config/custom-parameters/managementGroups.parameters.all.json"
         },
         {
-            "source": "infra-as-code/bicep/modules/mgDiagSettings/parameters/mgDiagSettings.parameters.all.json",
-            "destination": "config/custom-parameters/mgDiagSettings.parameters.all.json"
+            "source": "infra-as-code/bicep/orchestration/mgDiagSettingsAll/parameters/mgDiagSettingsAll.parameters.min.json",
+            "destination": "config/custom-parameters/mgDiagSettingsAll.parameters.all.json"
         },
         {
-            "source": "infra-as-code/bicep/modules/policy/assignments/parameters/policyAssignmentManagementGroup.dine.parameters.all.json",
-            "destination": "config/custom-parameters/policyAssignmentManagementGroup.dine.parameters.all.json"
+            "source": "infra-as-code/bicep/modules/policy/assignments/alzDefaults/parameters/alzDefaultPolicyAssignments.parameters.all.json",
```

Comment:
alzDefaultPolicyAssignments.parameters.all.json has quite a few xxxx-ed out values.  The log analytics workspace id is probably a straightforward one - but the Ddos and Dns based Ids might need some thought...  Presumably these are values from the connectivity subscription.